test/e2e/network/netpol/probe.go

```diff
@@ -108,7 +108,15 @@ func probeWorker(prober Prober, jobs <-chan *ProbeJob, results chan<- *ProbeJobR
 		// dnsName := job.PodTo.QualifiedServiceAddress(job.ToPodDNSDomain)
 
 		// TODO make this work on dual-stack clusters...
-		connected, command, err := prober.probeConnectivity(podFrom.Namespace, podFrom.Name, podFrom.Containers[0].Name(), job.PodTo.ServiceIP, job.Protocol, job.ToPort, timeoutSeconds)
+		connected, command, err := prober.probeConnectivity(&probeConnectivityArgs{
+			nsFrom:         podFrom.Namespace,
+			podFrom:        podFrom.Name,
+			containerFrom:  podFrom.Containers[0].Name(),
+			addrTo:         job.PodTo.QualifiedServiceAddress(job.ToPodDNSDomain),
```

Comment:
why do you change `job.PodTo.ServiceIP` to `job.PodTo.QualifiedServiceAddress(job.ToPodDNSDomain)` ?

refactors should not change current status, just reorganize the code, 
changes should be done in an independent commit so the reviewer can understand it , otherwise these things can be missed in reviews

Reply:
thanks for this catch @aojea  :) didnt see this PR until just now, but looks like its fixed